diff --git a/node_modules/@wagmi/cli/dist/plugins/index.d.ts b/node_modules/@wagmi/cli/dist/plugins/index.d.ts
index 04a78d1..0eb6430 100644
--- a/node_modules/@wagmi/cli/dist/plugins/index.d.ts
+++ b/node_modules/@wagmi/cli/dist/plugins/index.d.ts
@@ -119,7 +119,9 @@ declare const apiUrls: {
     137: string;
     80001: string;
     42161: string;
+    42170: string;
     421613: string;
+    421614: string;
     56: string;
     97: string;
     128: string;
diff --git a/node_modules/@wagmi/cli/dist/plugins/index.js b/node_modules/@wagmi/cli/dist/plugins/index.js
index 8e23da2..02b7c49 100644
--- a/node_modules/@wagmi/cli/dist/plugins/index.js
+++ b/node_modules/@wagmi/cli/dist/plugins/index.js
@@ -386,12 +386,12 @@ function blockExplorer({
         throw new Error(parsed.data.result);
       return parsed.data.result;
     },
-    request({ address }) {
+    request({ address, implementation }) {
       if (!address)
         throw new Error("address is required");
       return {
         url: `${baseUrl}?module=contract&action=getabi&address=${getAddress({
-          address
+          address, implementation
         })}${apiKey ? `&apikey=${apiKey}` : ""}`
       };
     }
@@ -1386,7 +1386,9 @@ var apiUrls = {
   [137]: "https://api.polygonscan.com/api",
   [80001]: "https://api-testnet.polygonscan.com/api",
   [42161]: "https://api.arbiscan.io/api",
+  [42170]: "https://api-nova.arbiscan.io/api",
   [421613]: "https://api-goerli.arbiscan.io/api",
+  [421614]: "https://api-sepolia.arbiscan.io/api",
   [56]: "https://api.bscscan.com/api",
   [97]: "https://api-testnet.bscscan.com/api",
   [128]: "https://api.hecoinfo.com/api",
@@ -1409,16 +1411,16 @@ function etherscan({
     address: typeof x.address === "string" ? { [chainId]: x.address } : x.address
   }));
   return blockExplorer({
-    apiKey,
-    baseUrl: apiUrls[chainId],
+    apiKey: `${apiKey}&chainid=${chainId}`,
+    baseUrl: `https://api.etherscan.io/v2/api`,
     cacheDuration,
     contracts,
-    getAddress({ address }) {
+    getAddress({ address, implementation = {} }) {
       if (!address)
         throw new Error("address is required");
       if (typeof address === "string")
         return address;
-      const contractAddress = address[chainId];
+      const contractAddress = implementation[chainId] ?? address[chainId];
       if (!contractAddress)
         throw new Error(
           `No address found for chainId "${chainId}". Make sure chainId "${chainId}" is set as an address.`
