diff --git a/node_modules/@wagmi/cli/dist/plugins/index.d.ts b/node_modules/@wagmi/cli/dist/plugins/index.d.ts
index 04a78d1..1574961 100644
--- a/node_modules/@wagmi/cli/dist/plugins/index.d.ts
+++ b/node_modules/@wagmi/cli/dist/plugins/index.d.ts
@@ -114,12 +114,15 @@ declare const apiUrls: {
     1: string;
     5: string;
     11155111: string;
+    17000: string;
     10: string;
     420: string;
     137: string;
     80001: string;
     42161: string;
+    42170: string;
     421613: string;
+    421614: string;
     56: string;
     97: string;
     128: string;
diff --git a/node_modules/@wagmi/cli/dist/plugins/index.js b/node_modules/@wagmi/cli/dist/plugins/index.js
index 8e23da2..4cc878a 100644
--- a/node_modules/@wagmi/cli/dist/plugins/index.js
+++ b/node_modules/@wagmi/cli/dist/plugins/index.js
@@ -1381,12 +1381,15 @@ var apiUrls = {
   [1]: "https://api.etherscan.io/api",
   [5]: "https://api-goerli.etherscan.io/api",
   [11155111]: "https://api-sepolia.etherscan.io/api",
+  [17000]: "https://api-holesky.etherscan.io/api",
   [10]: "https://api-optimistic.etherscan.io/api",
   [420]: "https://api-goerli-optimistic.etherscan.io/api",
   [137]: "https://api.polygonscan.com/api",
   [80001]: "https://api-testnet.polygonscan.com/api",
   [42161]: "https://api.arbiscan.io/api",
+  [42170]: "https://api-nova.arbiscan.io/api",
   [421613]: "https://api-goerli.arbiscan.io/api",
+  [421614]: "https://api-sepolia.arbiscan.io/api",
   [56]: "https://api.bscscan.com/api",
   [97]: "https://api-testnet.bscscan.com/api",
   [128]: "https://api.hecoinfo.com/api",
